views/standard/spec/columns.js

```diff
@@ -293,6 +293,13 @@ define(['jquery', 'appearanceUtils', 'underscore'], function($, utils, _) {
                    label.created_at, label.user);
                   node.append(icon);
                }
+               if ((label = pull.getLabel('external_block'))) {
+                  var icon = $('<i>').addClass('fa fa-eye-slash externally-blocked');
```

Comment:
Does the selector here actually work? Isn't `$('')` just for id selectors? 
I don't think the using angle brackets within that is valid. I think `$E('i')` might be what was intended here, but it's probably even better to select it by a specific class name in case there are multiple `i`s.